### PR TITLE
New chainIds

### PR DESCRIPTION
**Pull request by stackman27**

### CODE REVIEW

File: selectors.yml [hunk 1]
```diff
@@ -27,6 +27,9 @@ selectors:
   2358:
     selector: 5990477251245693094
     name: "ethereum-testnet-sepolia-kroma-1"
+  5001:
+    selector: 4168263376276232250
+    name: "ethereum-testnet-goerli-mantle-1"
```
Comment (andrevmatos) [inline selectors.yml:32]: goerli? still?

Comment (stackman27) [reply selectors.yml:32]: yea sepolia had issues so we're going to go with goerli + mainnet until the chain fixes them

File: selectors.yml [hunk 2]
```diff
@@ -27,12 +33,24 @@ selectors:
   2358:
     selector: 5990477251245693094
     name: "ethereum-testnet-sepolia-kroma-1"
+  4002: 
+    selector: 4905564228793744293
+    name: "fantom-testnet-1"
```
Comment (andrevmatos) [inline selectors.yml:38]: Hmm.. so far, the `-1` suffix is named `rollup_instance` in `Network naming proposal` doc, so I'm not sure it applies to testnets of other L1s, like this.
Is this a L1 testnet, or a rollup of other network?

File: selectors.yml [hunk 3]
```diff
@@ -27,12 +33,24 @@ selectors:
...
+    name: "ethereum-testnet-goerli-mantle-1"
   5003:
     selector: 8236463271206331221
     name: "ethereum-testnet-sepolia-mantle-1"
   43113:
     selector: 14767482510784806043
     name: "avalanche-testnet-fuji"
+  44787: 
+    selector: 3552045678561919002
+    name: "celo-testnet-alfajores-1"
```
Comment (andrevmatos) [inline selectors.yml:53]: Same

File: selectors.yml [hunk 4]
```diff
@@ -44,6 +62,9 @@ selectors:
   84532:
     selector: 10344971235874465080
     name: "ethereum-testnet-sepolia-base-1"
+  10200: 
+    selector: 8871595565390010547
+    name: "gnosis-testnet-chiado-1"
```
Comment (andrevmatos) [inline selectors.yml:67]: Same

File: selectors.yml [hunk 5]
```diff
@@ -53,12 +74,18 @@ selectors:
   534351:
     selector: 2279865765895943307
     name: "ethereum-testnet-sepolia-scroll-1"
+  595581: 
+    selector: 7837562506228496256
+    name: "avalanche-testnet-nexon-1"
```
Comment (andrevmatos) [inline selectors.yml:79]: Same

File: selectors.yml [hunk 6]
```diff
@@ -1,11 +1,17 @@
 selectors:
   # Testnets
+  31: 
+    selector: 8953668971247136127 
+    name: "bitcoin-testnet-rootstock-1"
```
Comment (andrevmatos) [inline selectors.yml:5]: Same

File: selectors.yml (hunk 2 above)
Comment (stackman27) [reply selectors.yml:38]: this is an L1 testnet 

https://docs.fantom.foundation/wallet/tutorials/set-up-metamask-testnet

Comment (stackman27) [reply selectors.yml:38]: I just keep it incase they release another testnet

Comment (andrevmatos) [reply selectors.yml:38]: If they release another testnet, it's expected that each has their own naming, in which case you'd go with `$l1_name-testnet-$testnet_name` instead

Comment (stackman27) [reply selectors.yml]: fixed! fantom doesnot have an explicit name for their testnet

File: selectors.yml (hunk 3 above)
Comment (stackman27) [reply selectors.yml]: fixed

File: selectors.yml (hunk 4 above)
Comment (stackman27) [reply selectors.yml]: fixed

File: selectors.yml (hunk 5 above)
Comment (stackman27) [reply selectors.yml]: fixed

File: selectors.yml (hunk 6 above)
Comment (stackman27) [reply selectors.yml]: fixed

File: selectors.yml [hunk 7]
```diff
@@ -44,6 +62,9 @@ selectors:
   84532:
     selector: 10344971235874465080
     name: "ethereum-testnet-sepolia-base-1"
+  10200: 
+    selector: 8871595565390010547
+    name: "gnosis-testnet-chiado"
```
Comment (andrevmatos) [inline selectors.yml:67]: Please, refer to [prior art](https://github.com/smartcontractkit/atlas/tree/master/postgres/migrations/evm/gnosis_chain-testnet-chiado) when addint any network which some product may already support!
```suggestion
    name: "gnosis_chain-testnet-chiado"
```

File: selectors.yml [hunk 8]
```diff
@@ -1,11 +1,17 @@
 selectors:
   # Testnets
+  31: 
+    selector: 8953668971247136127 
+    name: "bitcoin-testnet-rootstock"
```
Comment (andrevmatos) [inline selectors.yml:5]: Is this evm?

Comment (stackman27) [reply selectors.yml:5]: they have their own VM that is compatible with EVM

**Review by andrevmatos — APPROVED** [review]

**Pull request merged**